### ISSUE TITLE
No registered selector for ID: ms-vscode.npm-command

### ISSUE THREAD

**Issue by NeonWizard** [microsoft/vscode-js-debug#1859]
**Describe the bug**
If I open a new Javascript Debug Terminal and type `npm --help`, `npm --version`, or any other npm command, the terminal freezes for about 10 seconds and then I get the error "Debug adapter process has terminated unexpectedly (Extension host shut down)." Looking in the Window logs I see `[error] No registered selector for ID: ms-vscode.npm-command`.

**To Reproduce**
1. Click the dropdown on creating a new terminal, select "Javascript Debug Terminal".
2. In the new terminal, type "npm --version".
3. Wait about 10 seconds, you should see the extension host crash.

**Log File**

```
2023-10-24 16:28:25.853 [error] No registered selector for ID: ms-vscode.npm-command: Error: No registered selector for ID: ms-vscode.npm-command
    at vscode-file://vscode-app/Applications/Visual%20Studio%20Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:1535:26336
```

**VS Code Version:** v1.83.1 (Universal)

**Additional context**
Oddly, this issue doesn't affect my coworker, who has a nearly identical environment to mine. We have the same laptop (M1 Macbook running Ventura), on the same VSCode version, with the same installation of node and npm. I've tried clearing out all of my VSCode settings and disabling all extensions. I've reinstalled node and npm multiple times.

**Comment by NeonWizard** [comment]
Found the root cause I think. I discovered this issue only happens when VSCode has my project directory open - any other directories are fine. I have a massive folder (4 gigabytes) somewhere in the project folders containing logs from test runs. When I removed the folder, the issue disappeared. I have no idea how running `npm --version` correlates to directory size. Running `npm --version` outside of the Javascript Debug Terminal works fine with no latency.
